utils/utils.go

```diff
@@ -284,23 +286,66 @@ func BuildTargetPath(pattern, path, target string, ignoreRepo bool) (string, boo
 
 	groups := r.FindStringSubmatch(path)
 	if len(groups) > 0 {
-		target, replaceOccurred := ReplacePlaceHolders(groups, target)
+		target, replaceOccurred, err := ReplacePlaceHolders(groups, target, isRegexp)
+		if err != nil {
+			return "", false, err
+		}
 		return target, replaceOccurred, nil
 	}
 	return target, false, nil
 }
 
-// group - regular expression matched group to replace with placeholders
+// ReplacePlaceHolders group - regular expression matched group to replace with placeholders
 // toReplace - target pattern to replace
+// isRegexp -
```

Comment:
Missing argument description.

Also, the arguments and their description should be placed after the function's description, and not before it.